packages/-ember-data/src/version.ts

```diff
@@ -1,4 +1 @@
-import { getOwnConfig } from '@embroider/macros';
-
-const VERSION = getOwnConfig<{ VERSION: string }>().VERSION;
-export default VERSION;
+export { version as default } from '../package.json';
```

Comment:
is this guaranteed to tree-shake? Or does it pull the full json object into the built asset?

Reply:
it replaces with the string value -- doesn't even keep the import in the output

Reply:
```sh
$ cat dist/version.js                                                                                                        [version-inlining][ruby-2.7.4]
const version = "5.4.0-alpha.88";
export { version as default }             
```

Reply:
Nice, then we should be able to merge this as-is. The mirror publish step uses a regex to replace package names where-ever they appear and the version would match since they are always in sync.